src/redis.rs

```diff
@@ -42,6 +43,22 @@ impl RedisActor {
 
         Supervisor::start(|_| RedisActor {
             addr,
+            db: None,
+            cell: None,
+            backoff: backoff,
+            queue: VecDeque::new(),
+        })
+    }
+
+    pub fn start_db<S: Into<String>>(addr: S, db: u8) -> Addr<RedisActor> {
```

Comment:
Maybe instead of having separate `start_db` with db index, it would be better have builder-styled method to set index?

`fn set_db(mut self, db: u8) -> Self`